linker-common-lib/src/main/java/com/linker/common/network/SocketClientHandler.java

```diff
@@ -0,0 +1,44 @@
+package com.linker.common.network;
+
+import com.linker.common.MessageContent;
+import com.linker.common.MessageContentOutput;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.SimpleChannelInboundHandler;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class SocketClientHandler extends SimpleChannelInboundHandler<MessageContentOutput> {
```

Comment:
Shouldn't these be `@Component`?

Reply:
one handler is one layer of netty message processing stack, and each connection has it's own handler which can particular data of current channel,  then handler is designed to be shared, but @Component is singleton, which can not save private data